ddR/R/ddR.R

```diff
@@ -80,32 +80,40 @@ useBackend <- function(driver, ...) {
 #' @export
 setClass("ddRDriver", representation(DListClass = "character", DFrameClass = "character", DArrayClass = "character", backendName = "character"))
 
-#' Called when the backend driver is initialized.
+#’ @rdname package-deprecated
+#' Initialize backend driver
 #'
-#' @param x The driver object to initialize the backend for.
+#' Used internally by ddR to set up a new backend driver.
+#'
+#' @param driver The driver object to initialize the backend for.
 #' @param ... Other parameters to pass to the initialization routine.
+#' @seealso \code{\link{useBackend}} for the user facing way to set or
+#'      change backends
 #' @export
-setGeneric("init", function(x,...) standardGeneric("init"))
+setGeneric("init", function(driver,...) standardGeneric("init"))
```

Comment:
Many generics in R use the `x` convention. While it would be fine to use `driver` for the generics we define, there is going to be inconsistency with other generics. 

A related problem is sharing generics with other popular packages. For example, Bioconductor defines the `combine(x, y, ...)` generic. We should try to avoid conflict if at all possible.